src/SFA.DAS.CommitmentPayments.WebJob.UnitTests/app.config

```diff
@@ -50,6 +50,10 @@
         <assemblyIdentity name="Microsoft.IdentityModel.Clients.ActiveDirectory" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-3.18.0.2306" newVersion="3.18.0.2306" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.0.0.0" />
```

Comment:
Why bind version 4.2 to 4.0? Shouldn't we be using 4.2

Reply:
System.Net.Http has a version in the .net framework and also a version in a package. This oddity, binding <=4.2 to 4.0, causes the .net framework version to be loaded as a module, as opposed to the package version. (The package version is no longer being updated.)
We still need the System.Net.Http.4.3.3 package, as it is declared as a dependency to one of the packages we had to update (SFA.DAS.Apprenticeships.Api.Client I think).
The System.Net.Http issue has been fixed in .net 4.7.1 - it always uses the framework version irrespective of the reference or redirect, but cloud services doesn't support 4.7 out of the box.
We'll need to be vigilant from now on, as the nuget package manager tries to rewrite the binding when adding packages, which we'll then have to manually undo.